lib/tasks/whitehall_importer.rb

```diff
@@ -62,17 +66,24 @@ def most_recent_edition
     end
 
     def create_or_update_document
-      Document.find_or_create_by(
+      first_edition = whitehall_document["editions"].first
+      first_author = first_edition["history"].select { |h| h["event"] == "create" }.first
+
+      Document.find_or_create_by!(
         content_id: whitehall_document["content_id"],
         locale: "en",
         document_type_id: "news_story", ## To be updated once Whitehall exports this value
         created_at: whitehall_document["created_at"],
         updated_at: whitehall_document["updated_at"],
+        created_by_id: user_map.select { |u| u["whitehall_user_id"] == first_author["whodunnit"] }.first["content_publisher_user_id"],
         imported_from: "whitehall",
       )
     end
 
     def create_edition(document, translation, whitehall_edition, edition_number)
+      first_author = whitehall_edition["history"].select { |h| h["event"] == "create" }.first
+      last_author = whitehall_edition["history"].last
```

Comment:
Do we know if they're always an editor? Looking up events in Whitehall we get 3 used ones so I think this is probably safe.

```
irb(main):003:0> Version.distinct.pluck(:event)
=> ["create", "update", "destroy"]
```